action.bash

```diff
@@ -33,21 +33,20 @@ push () {
 	fi
 }
 
+re_docker_tag="^refs/(heads|tags)/([^/]+|[a-z]+/(.*))$"
+
 case "$GITHUB_EVENT_NAME" in
-	pull_request)
-		grep -qEe '^refs/pull/[0-9]+' <<<"$GITHUB_REF"
-		TAG="pr$(grep -oEe '[0-9]+' <<<"$GITHUB_REF")"
-		mkimg
-		;;
-	push)
-		grep -qEe '^refs/heads/.' <<<"$GITHUB_REF"
-		TAG="$(cut -d / -f 3- <<<"$GITHUB_REF")"
-		mkimg "$TAG"
-		push
-		;;
+	workflow_dispatch)
+		;&
+	schedule)
+		;&
```

Comment:
What's `;&`?

Reply:
>        case word in [ [(] pattern [ | pattern ] ... ) list ;; ] ... esac
>              A case command first expands word, and tries to match it against each pattern in turn, using the matching rules described under Pattern Matching below.  The word is expanded using tilde expansion, parameter and variable expansion, arithmetic  expansion,
>              command substitution, process substitution and quote removal.  Each pattern examined is expanded using tilde expansion, parameter and variable expansion, arithmetic expansion, command substitution, and process substitution.  If the nocasematch shell op‐
>              tion is enabled, the match is performed without regard to the case of alphabetic characters.  When a match is found, the corresponding list is executed.  If the ;; operator is used, no subsequent matches are attempted after the first pattern match.  Us‐
>              ing ;& in place of ;; causes execution to continue with the list associated with the next set of patterns.  Using ;;& in place of ;; causes the shell to test the next pattern list in the statement, if any, and execute any associated list on a successful
>              match.  The exit status is zero if no pattern matches.  Otherwise, it is the exit status of the last command executed in list.

Reply:
Why not `workflow_dispatch|schedule|release)`?

Reply:
Doesn't it work?

Reply:
That's not the point, nor answers my question.

Reply:
There's no answer, both options are equal.

Reply:
But mine's more readable.

Reply:
... for me.

Reply:
Then feel free to change it

Reply:
Sure. And the next time something's wrong with an Icinga Web PR of mine, I'll say you "feel free to change it".`</irony>`

Reply:
Well, no. There's nothing *wrong* here, it's an alternative syntax and not even a style related thing, that's why I don't see a point of changing this.

Reply:
> not even a style related thing

I don't agree. Mine is a more not-a-bash-pro-friendly **style**.

I've googled "bash case" and scrolled through the first four tutorials. Two of them uses my style and zero are using yours. Keep in mind that code is read more often than written.